### PR TITLE
Created view and explore files to add more steps in funnel analysis 

### PR DESCRIPTION
**Pull request by YeonjooSmith**
Created view and explore files to add more steps in funnel analysis for mobile vpn glean events. 
Expansion from "/looker-hub/mozilla_vpn/views/funnel_analysis.view.lkml"

### CODE REVIEW

File: mozilla_vpn/explores/funnel_analysis_granular_steps.explore.lkml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+include: "/spoke-default/mozilla_vpn/views/funnel_analysis_granular_steps.view.lkml"
```
Comment (birdsarah) [inline mozilla_vpn/explores/funnel_analysis_granular_steps.explore.lkml:1]: Thinking through naming - what about `funnel_analysis_detailed` or `funnel_analysis_extra_steps`.

......

**Comment by birdsarah** [comment]
Thanks for putting this together @YeonjooSmith. For context for whoever's reviewing. We have a lot of multi-step events in VPN and being able to do this kind of analysis is extremely helpful.

File: mozilla_vpn/explores/funnel_analysis_granular_steps.explore.lkml (hunk 1 above)
Comment (YeonjooSmith) [reply mozilla_vpn/explores/funnel_analysis_granular_steps.explore.lkml]: Changed the file names to funnel_analysis_extra_steps and committed the changes.

**Review by relud — APPROVED** [review]

**Comment by YeonjooSmith** [comment]
@relud Thank you so much for reviewing and approving the PR fast! I really appreciate it!!!

**Pull request merged**